bindata/cert-manager-deployment/webhook/cert-manager-webhook-deployment.yaml

```diff
@@ -74,6 +74,7 @@ spec:
             capabilities:
               drop:
                 - ALL
+      enableServiceLinks: false
```

Comment:
Can you check if the previous default was "true"? If so we'll need to call out under release notes.

Reply:
Found the upstream PR that introduced this change, https://github.com/cert-manager/cert-manager/pull/6292. However, it looks like this was added for some Helm customization and is unlikely to affect the default (k8s defaults that value to true), but it does explicitly set to false here now.

release-1.12 branch upstream doesn't contain this field at all, i.e. uses k8s default true.

Reply:
CC @snarayan-redhat as a tracker reminder into `https://issues.redhat.com/browse/OSDOCS-8126`